src/main/java/edu/texas/social_computing/hospitals/Matching.java

```diff
@@ -4,12 +4,11 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Multimap;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
 
 public class Matching {
 
-    private static final Hospital NO_MATCH = Hospital.create("NO_MATCH", -1, 0, ImmutableList.of());
+    static final Hospital NO_MATCH = Hospital.create("NO_MATCH", -1, 0, ImmutableList.of());
```

Comment:
just curious, why make this non-private?

Reply:
cuz it was late at night. ill change it back

Reply:
:P